presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -68,6 +84,19 @@ private static TableDefinition partitionedTableDefinition()
                 .build();
     }
 
+    private static TableDefinition partitionedTableWithVariablePartitionsDefinition()
+    {
+        String tableName = "partitioned_table_with_too_many_partitions";
+        String createTableDdl = "CREATE TABLE %NAME%(col INT) " +
+                "PARTITIONED BY (part_col INT) " +
+                "STORED AS ORC";
+
+        return HiveTableDefinition.builder(tableName)
+                .setCreateTableDDLTemplate(createTableDdl)
+                .setNoData()
```

Comment:
instead of writing custom utility to create partition I would prefer to use `addPartition` here.

Reply:
I can't, because it takes a lot of time (>30s) and is re-done before every test method. See TODO note in `createPartitions(String tableName, int partitionsToCreate)`

Reply:
I see. Anyway it would be great to improve tempto. There are two issues: 
 - add support partitions to immutable
 - make partitions to be created faster

We could do it as follow up.